src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/components/AuthContent/index.js

```diff
@@ -2,8 +2,24 @@
 import React from 'react';
 // Relative imports.
 import CallToActionWidget from 'platform/site-wide/cta-widget';
+import CernerCallToAction from '../../../components/CernerCallToAction';
 import MoreInfoAboutBenefits from '../../../components/MoreInfoAboutBenefits';
 
+const callToActions = [
+  {
+    deriveHeaderText: facilityNames =>
+      `Manage appointments with ${facilityNames}`,
+    href: '',
+    label: 'Go to My VA Health',
+  },
+  {
+    deriveHeaderText: () =>
+      `Manage appointments at all other VA medical centers`,
+    href: '',
```

Comment:
Need the 2 `href` links [here](https://adhoc.invisionapp.com/share/DYVMI5OXEQN#/screens/404881778):

![image](https://user-images.githubusercontent.com/12773166/86203952-86eab680-bb23-11ea-98fd-e164f6bfab9e.png)

Reply:
**From Cerner, for August launch of this specific tool page update:**

Centralized Scheduling Solution (CSS): Veterans at Chalmers P. Wylie VA Ambulatory Care Center, Columbus OH (757) will utilize scheduling only functionality. My VA Health will be contextually aware of the Veteran and only present the following functionality:
View, book, request, and cancel appointments online  
Manage appointments: Add to Calendar, View Instructions, Print Details
Appointments booked online for VA Medical Centers converted to CSS are visible for staff in Cerner and in VistA Scheduling
VA.gov: Schedule and View Appointments
Link to this in Staging My VA Health: https://ehrm-va-test.patientportal.us.healtheintent.com/pages/scheduling/upcoming
Link to this in PROD My VA Health: https://patientportal.myhealth.va.gov/pages/scheduling/upcoming

**From Cerner, for later launch of Mann-Grandstaff updates as well:**

Electronic Health Record Modernization (EHRM) Spokane Mann Grandstaff IOC: Veterans receiving care at VA Medical Centers who have converted to EHRM will have the following functionality:
View, direct book, cancel, request, and manage appointments (Scheduling)
VA.gov: Schedule and View Appointments
Link to this in Staging My VA Health: https://ehrm-va-test.patientportal.us.healtheintent.com/pages/scheduling/upcoming
Link to this in PROD My VA Health: https://patientportal.myhealth.va.gov/pages/scheduling/upcoming